ext/rp/irq.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, Andrey Kunitsyn
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+/**
+ * Pico SDK compatibility for tinyusb
+ */
+#include "hardware/regs/intctrl.h"
+
+static void irq_set_enabled(int irqn,bool enable) {
```

Comment:
```suggestion
static inline void irq_set_enabled(int irqn,bool enable) {
```
Or does static already imply inline in C? In C++ at least this can give a linker error.